documentation/modules/overview/con-overview-components-topic-operator.adoc

```diff
@@ -10,7 +10,7 @@ The Topic Operator provides a way of managing topics in a Kafka cluster through
 
 .Example architecture for the Topic Operator
 
-image:topic-operator.png[Topic Operator]
+image:topic-operator.png[The Topic Operator manages topics for Kafka and ZooKeeper clusters via KafkaTopic resources]
```

Comment:
"manages topics for the Zookeeper cluster"? I am not sure what that means? Topics don't apply to the ZK cluster?